media/src/App.svelte

```diff
@@ -790,6 +794,9 @@
             <i class="bi bi-person-fill"></i>
             {activeUserCount}
         </div>
+        <button class="leave-room" onclick="window.location.href='/'" title="Exit Room">
```

Comment:
This would be more standards and accessibility-friendly if it was an `<a href="` tag instead of a `<button>` element with a custom onclick event. You can style an `<a>` tag just like a button, and that's commonly done in bootstrap for button elements that simply navigate to a new URL.